integration_tests/__tests__/snapshot-test.js

```diff
@@ -41,4 +49,74 @@ describe('Snapshot', () => {
       output['snapshot is not influenced by previous counter 0']
     ).not.toBe(undefined);
   });
+
+  describe('validates existance of snapshots correctly', () => {
+    const pathToOriginal = path.resolve(
+      __dirname,
+      '../snapshot/__tests__/snapshot.js'
+    );
+    const originalContent = String(fs.readFileSync(pathToOriginal));
+    const pathToCopy = pathToOriginal.replace('.js', '_copy.js');
```

Comment:
let's construct this without `.replace`:

```js
const testsFolder = path.resolve(__dirname, '../snapshot/__tests__');
const original = fs.readFileSync(path.join(testsFolder, 'snapshot.js'));
const copy = path.join(testsFolder, 'snapshot_copy.js');
```

Also, let's do this outside of `describe` either in a `beforeEach` or on the top level :)